features/reforger_upload.feature

```diff
@@ -13,6 +13,15 @@ Scenario: Upload next mission
   And Zeus specifies <modlist.json>, <scenarioId>, <filename>
   Then a new server config file is created
   And the config file is patched with <modlist.json> and <scenarioId>
+  And the version numbers are removed from the mods.
```

Comment:
Sounds like we should flip around the below case as the default = Normal case is to keep versions, this one is now the second scenario (first variant = strip version numbers)

Reply:
I couldn't quite wrap my head around how to do this properly, so split into a new issue to not block the merge.